docs/smtchecker.rst

```diff
@@ -448,10 +448,12 @@ the JSON option ``settings.modelChecker.timeout=<time>``, where 0 means no timeo
 Verification Targets
 ====================
 
-The types of verification targets created by the SMTChecker can also be customized via
-the CLI option ``--model-checker-target <targets>`` or the JSON option
-``settings.modelChecker.targets=<targets>``, where ``<targets>`` is a no-space-comma-separated
-list of one or more verification targets. The keywords that represent the targets are:
+The types of verification targets created by the SMTChecker can also be
+customized via the CLI option ``--model-checker-target <targets>`` or the JSON
+option ``settings.modelChecker.targets=<targets>``, where ``<targets>`` is a
+no-space-comma-separated list of one or more verification targets for the
+former, and an array of zero or more targets as strings in the latter.
+The keywords that represent the targets are:
```

Comment:
Maybe this should be more clear if we split up instead of forming such long sentence? Probably people will end up reading it twice like I did when first started review